packages/mobile/src/components/drawer/DrawerHeader.tsx

```diff
@@ -59,14 +70,17 @@ export const DrawerHeader = (props: DrawerHeaderProps) => {
           onPress={onClose}
           style={styles.dismissContainer}
         >
-          <IconRemove width={30} height={30} fill={closeColor} />
+          <IconRemove width={30} height={30} fill={iconRemoveColor} />
         </TouchableOpacity>
       )}
       {title && (
         <View style={styles.titleContainer}>
-          {titleIcon && (
-            <Image style={styles.titleIcon as ImageStyle} source={titleIcon} />
-          )}
+          {TitleIcon ? (
```

Comment:
Really wanted to get this to be a `ImageSrcProp | ComponentType<SvgProps>` but was not able to disambiguate the type when it came time for usage

Reply:
I think this is ok... except I wish it took the same shape. Two args is fine and pretty good devx, but maybe image should be a component rather than imageprops